.github/workflows/test.yml

```diff
@@ -60,6 +63,75 @@ jobs:
           pip install fabio==0.12
       if: ${{ matrix.config.os == 'macos-latest'}}
 
+    - name: Install Eigen on Linux
+      run: |
+          sudo apt-get update
+          sudo apt-get install -y libeigen3-dev
+      if: runner.os == 'Linux'
+
+    - name: Install Eigen on MacOS
+      run: |
+          brew update
+          brew install eigen
+      if: runner.os == 'macOS'
+
+    - name: Install Eigen on Windows
+      run: |
+          Invoke-WebRequest -Uri https://gitlab.com/libeigen/eigen/-/archive/3.4.0/eigen-3.4.0.zip -o eigen.zip
+          Expand-Archive -Path eigen.zip -DestinationPath C:\eigen3
+          cp -r C:\eigen3\eigen-3.4.0 D:/a/hexrd/hexrd/eigen
+      if: runner.os == 'Windows'
+      shell: pwsh
+
+    - name: Install xsimd on Linux
+      run: |
+          sudo apt-get update
+          sudo apt-get install -y cmake g++ wget unzip
+          wget -O xsimd.tar.gz https://github.com/xtensor-stack/xsimd/archive/refs/tags/7.4.10.tar.gz
+          tar -xf xsimd.tar.gz
+          cd xsimd-7.4.10
+          mkdir build
+          cd build
+          cmake ..
+          make -j$(nproc)
+          sudo make install
```

Comment:
We might want to switch the `test.yml` workflow over to use a conda development installation instead of a pip development installation, especially if we decide to stop supporting pip development installations, so that the extra step of installing/compiling dependencies beforehand is not required.

Reply:
I'd love to discuss modifications to our current workflow in a call, if you're available!